scripts/tortoisesvn/tortoisesvn-install-tools.reg

```diff
@@ -0,0 +1,20 @@
+REGEDIT4
+
+[HKEY_CURRENT_USER\Software\TortoiseSVN\DiffTools]
+".doc"="wscript.exe \"C:\\TortoiseSVN\\Diff-Scripts\\diff-doc.js\" %base %mine //E:javascript"
```

Comment:
Should these be %TSVNHOME% instead of the hardcoded C:\\TortoiseSVN?

Reply:
Yes, I totally missed those, thanks. Fixed now,